helm-build-push/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: Helm build and push chart
+description: Push helm chart to OCI-based registries
+inputs:
+  CHARTS_PATH:
+    description: Path to charts base folder
+    default: ./charts
+    required: false
+  CHART_NAME:
+    description: "Chart to build and push"
+    required: true
+  REGISTRY:
+    description: Registry
+    default: ghcr.io
```

Comment:
```suggestion
    default: ${{ vars.IMAGE_REGISTRY }}
```

Reply:
You cannot use repository vars in actions

Reply:
I use them too, and this is a global variable.
[This](https://github.com/greenbone/actions/blob/main/lock-branch/action.yaml#L15) for example is working.

Did you test it?